i18n/locales/en.ts

```diff
@@ -2969,16 +2980,18 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
       title: `Make another payment`,
       nav_title: `Make another payment`,
       heading_one: `Make another payment`,
-      paragraph_one: `The night is still young, and you're not flying your Budgetcopter home. Got time for another beer? Now that you and Laszlo have your asymmetrical transactions and a flow worked out with revocation keys, let's run the protocol a few more times.`,
-      paragraph_two: `The two transactions representing the current state are on the screen, where you have paid Laszlo 1,000 satoshis. Send another 1,000 satoshi payment to Laszlo.`,
-      list_one: 'Update the amounts and output scripts',
+      paragraph_one: `The night is still young, and you're not flying your Budgetcopter home. Got time for another beer? Now that you and Laszlo have your asymmetric transactions and a flow worked out with revocation keys, let's run the protocol again.`,
+      paragraph_two: `The two commitment transactions representing the current state where you have paid Laszlo 1,000 sats for a beer are on your ePhone Infinity screen.`,
+      heading_two: `Instructions`,
+      list_one: `Update the amounts and output scripts for Laszlo's new commitment transaction`,
       list_two: "Sign Laszlo's transaction and send it to him",
-      list_three: `Laszlo will revoke his last state by sending you <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base whitespace-nowrap">REVOCATION_LASZLO_2</span>`,
+      list_three: `Update your commitment transaction: Laszlo will revoke his last state by sending you <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base whitespace-nowrap">REVOCATION_LASZLO_1</span>. Update the amounts and output scripts in your commitment transaction.`,
```

Comment:
wait is this the second revocation key or the first?

Reply:
It's the first. He gives you his previous revocation key, `REVOCATION_LASZLO_1` to prove that he won't try to settle/close the payment channel with the previous state. You might be getting thrown off because he has no reason to broadcast the prior commitment transaction. The newer state where he has 2,000 sats is more favorable to him.

I think this set of instructions could use some improvements (possibly in a follow up PR after the release) but the change I made here is just for correctness.